Sandbox/Sandbox.h

```diff
@@ -2,6 +2,41 @@
 
 #include <Aquarius.h>
 
+#include <array>
```

Comment:
I know we are using this for testing right now but thinking ahead when we want people to be able to include the single Aquarius header file should we put `#include<array>`in `Aquarius.h`? That way people do not need to include it in their Sandbox as I am thinking arrays will be used in pretty much any situation as you have done below. It is not a big deal but I am just thinking ahead.